src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -4041,10 +4041,10 @@ func main() {
     // TODO: Handle error.
   }
 
-  // Project ID for this request.
+  // * Project ID for this request.
   project := "" // TODO: Update placeholder value.
 
-  // The name of the zone where the instance group is located.
+  // * The name of the zone where the instance group is located.
```

Comment:
Could you add to this thread a link to the github issue where the asterisks will be fixed?